obal/data/roles/build_package/tasks/koji.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: "Confirm package is whitelisted"
+  package_whitelist_check:
+    releasers_conf: "{{ inventory_dir }}/rel-eng/releasers.conf"
```

Comment:
Why do we need a releasers.conf if we have releasers specified?

Reply:
I need both the given releaser and `releasers.conf` so i can figuring out how a releaser is mapped to a tag.